content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md

```diff
@@ -70,6 +70,21 @@ Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice f
 * Black Hole telemetry for TCP and HTTP protocols is not supported.
 * Histogram buckets are [significantly different](https://github.com/istio/istio/issues/20483) than Mixer Telemetry and cannot be changed.
 
+## Traffic management resource visibility changes
+
+In Istio 1.5 proxy configuration for hosts is determined by [`VirtualService`](/docs/reference/config/networking/virtual-service) visibility in
+addition to that of any relevant [`ServiceEntry`](/docs/reference/config/networking/service-entry/).
+
+If in previous versions you relied on [`Sidecar`](/docs/reference/config/networking/sidecar/) resources to 
+restrict visibility of hosts (mesh internal or external) to a target set of sidecar proxies, you now also need to 
+consider the hosts implied by any [`VirtualService`](/docs/reference/config/networking/virtual-service)s.
+
+Depending on your use of [`Sidecar`](/docs/reference/config/networking/sidecar/) resources in your mesh, this may require 
+you to review the namespaces that your [`VirtualService`](/docs/reference/config/networking/virtual-service)s are in to
```

Comment:
```suggestion
you to review the namespaces that your [`VirtualService`](/docs/reference/config/networking/virtual-service) are in to
```

Reply:
The previous pluralisation seems correct to me - otherwise the _are_ afterwards doesn't seem to read properly, and changing it to _is_ implies you might only have a single `VirtualService`